### ISSUE TITLE
Escaping issue

### ISSUE THREAD

**Issue by Typositoire** [influxdata/influxdb-ruby#115]
I've ran into an issue where I get this

```
unable to parse 'apachecombined-access clientip=\"X.X.X.X\",request=\"/XXX/XXX/99999/XXX/1?XXXXX=[\\\\\"XXXXX\\\\\"]&XXXX=0\",response.integer=200,bytes.long=703,response_time.double=0.065,time=1443528858': invalid number\n
```

The original event request was like this (I could not find the EXACT same but I've got enough to be sure there are all the same) 

```
/XXX/XXXX/99999/XXXX/1?XXXX=[\"XXXX\"]&XXXX=0
```

I guess the already escaped " caused an issue and made influxdb think the IP was not surrounded by quotes. 

```ruby
~|⇒  irb
irb(main):001:0> def escape_value(value, quote_escape)
irb(main):002:1>       val = value.
irb(main):003:1*         gsub(/\s/, '\ ').
irb(main):004:1*         gsub(',', '\,').
irb(main):005:1*         gsub('"', '\"')
irb(main):006:1>       val = %("#{val}") if quote_escape
irb(main):007:1>       val
irb(main):008:1>     end
=> nil
irb(main):009:0> escape_value('/XXXX/XXXXX/99999/XXXX/1?XXXX=[\"XXXX\"]&XXXXX=0', true)
=> "\"/XXXX/XXXX/99999/XXXX/1?XXXXX=[\\\\\"XXXX\\\\\"]&XXXX=0\""
```

**Comment by yhara** [comment]
I got the same error. This patch to `lib/influxdb/point_value.rb` fixed it.

```diff
     def escape_value(value, quote_escape)
       val = value.
+        gsub("\\"){ "\\\\" }.
         gsub(/\s/, '\ ').
         gsub(',', '\,').
         gsub('"', '\"')
```

**Comment by toddboom** [comment]
This should be fixed in #121, I believe. Thanks!
